extensions/postgis/Makefile.in

```diff
@@ -51,7 +51,9 @@ EXTENSION_UPGRADE_SCRIPTS = \
 	sql/raster_unpackage.sql \
 	sql/postgis_upgrade.sql \
 	../../doc/postgis_comments.sql \
-	../postgis_extension_helper_uninstall.sql
+	../postgis_extension_helper_uninstall.sql \
```

Comment:
What if you move this line after the spatial_ref_sys lines to see if it fixes the unpackaged issue.

Reply:
I have no idea what that changes, but I'm game for anything!

Reply:
Arg, drone is unconveniently busted right now https://cloud.drone.io/postgis/postgis/2710